NineChronicles.Headless/GraphTypes/FungibleAssetValueInputType.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Numerics;
+using GraphQL.Types;
+using Libplanet;
+using Libplanet.Assets;
+using Libplanet.Explorer.GraphTypes;
+
+namespace NineChronicles.Headless.GraphTypes
+{
+    public class FungibleAssetValueInputType : InputObjectGraphType<FungibleAssetValue>
+    {
+        public FungibleAssetValueInputType()
+        {
+            Field<NonNullGraphType<BigIntGraphType>>("quantity");
+            Field<NonNullGraphType<StringGraphType>>("ticker");
+            Field<NonNullGraphType<ByteGraphType>>("decimalPlaces");
+            Field<ListGraphType<AddressType>>("minters");
```

Comment:
This type seems able to represent:

- `null`
- `[]`
- `[null, null]`
- `[ADDRESS_A, null, ADDRESS_B]`

Were the third and the fourth also intended?

Reply:
no, i'll fix it.